src/loaders/lottie/tvgLottieModel.cpp

```diff
@@ -141,7 +141,11 @@ void LottieImage::prepare()
 void LottieTrimpath::segment(float frameNo, float& start, float& end, LottieExpressions* exps)
 {
     start = this->start(frameNo, exps) * 0.01f;
+    if (start > 1.0f) start = 1.0f;
```

Comment:
Suggestion:
```
if () ...
else if() ...
```
Otherwise, probably we can introduce clamp() method in math general functions.
```
template <typename T>
static inline void clamp(T& v, T& min, T& max)
{
    if (v < min) v = min;
    else if (v > max) v = max;
}
```